src/css/how-to-place.css

```
@@ -0,0 +1,88 @@
+#content-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin: 2vh 1vw;
+}
+
+#title-wrapper {
+    margin: 1vh 1vw;
+    padding: 4vh 2vw;
+    border-style: solid;
+    border-width: 0.1vh;
+    border-color: #3f3f3f;
+    border-radius: 0.5rem;
+    background-color: #dddddd;
+    text-align:center;
+}
+
+h1 {
+    font-size: 2rem;
+}
+
+#page-info-container {
+    display: flex;
+    justify-content: space-between;
+}
+
+#page-references {
+    display: flex;
+    justify-content: space-between;
+}
+
+#page-info-container article {
+    flex-basis: 50%;
+    margin: 1vh 1vw;
+    border-style: solid;
+    border-width: 0.1vh;
+    border-color: #3f3f3f;
+    border-radius: 0.5rem;
+    background-color: #dddddd;
+}
+
+#page-references article {
+    flex-basis: 100%;
+    margin: 1vh 1vw;
+    border-style: solid;
+    border-width: 0.1vh;
+    border-color: #3f3f3f;
+    border-radius: 0.5rem;
+    background-color: #dddddd;
+}
+
+#ekg-pic img{
+    display:block;
+    border-radius: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-bottom: 10%;
+}
+
+#page-info-container header {
+    padding: 1vh 2vw 1vh 2vw;
+    text-align: center;
+    font-size: 140%;
+}
+
+#page-references header {
+    padding: 1vh 2vw 1vh 2vw;
+    text-align: center;
+    font-size: 140%;
+}
+

```

This should help with the spacing of the references.
```suggestion

#page-references article {
    padding: 1vh 2vw 2.5vh 2vw;
}

```